### ISSUE TITLE
Provide changelogs for releases

### ISSUE THREAD

**Issue by Drugoy** [rootkiwi/an2linuxclient#40]
Please, provide changelogs for releases.

**Comment by rootkiwi** [comment]
Yeah, when I have time I have a few things that I plan to do for AN2Linux. For changelogs improvements can be made, at the moment changelogs is duplicated here:
https://github.com/rootkiwi/an2linuxclient/tree/master/fastlane/metadata/android/en-US/changelogs
and here:
https://github.com/rootkiwi/an2linuxclient/blob/master/app/src/main/res/values/changelog.xml

**Comment by Drugoy** [comment]
Oh, I didn't really mean to add them to the application itself, I just failed to find them at all.
Since you DO have changelogs (they are just not so easy to find), couldn't you copy that text to [GitHub releases messages](https://github.com/rootkiwi/an2linuxclient/releases)?

People need to be able to get new version's changelog BEFORE installing a new version.
I read changelogs to decide whether it's worth updating or not.

**Comment by rootkiwi** [comment]
I see. Agree, reading changelogs before installing should be more straightforward.
Yes I could do that. Really no reason why I haven't done that.
